backend/ml_model/utility/model_util.py

```diff
@@ -0,0 +1,90 @@
+"""This file will hold the helper functions for the ml-model folder"""
```

Comment:
Yeah this utilities file keeps the cleaning and miscelanous functions. They were originally mainly in model so I guess it fits with S in SOLID since now model.py only has one purpose. Clean architecture wise the we use by layer architecture so it fits with that too (having a utility folder)